bigbluebutton-html5/imports/api/users/server/methods/setRandomUser.js

```diff
@@ -17,8 +17,15 @@ export default function setRandomUser() {
 
     const payload = {
       requestedBy: requesterUserId,
+      allowRepeat: false
     };
 
+    if(allowrepeat){
```

Comment:
From now the message requires `allowRepeat` - `case class SelectRandomViewerReqMsgBody(requestedBy: String, allowRepeat: Boolean)` you should pass it in the json regardless of true or false value, but should ensure the value is a boolen type